CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Cromwell Change Log
 
+## 54 Release Notes
+
+### Per-backend hog factors
+Cromwell now allows overriding system-level log factors on back-end level. First, Cromwell will try to use hog-factor 
+defined in the backend config, and if it is not defined, it will default to using system-wide hog factor.
```

Comment:
We should also update the readthedocs page on hog factors (https://cromwell.readthedocs.io/en/develop/cromwell_features/HogFactors/#hog-limit), and link to it from here for context (in case people don't know what a hog factor is)